deep-learning/_posts/2019-05-09-PPO.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: review
+title:  "Clipped Proximal Policy Optimization Algorithm"
+tags:   deep-learning
+author: Antoine Théberge
+pdf:    "https://arxiv.org/pdf/1707.06347"
+cite:
+    authors: "John Schulman, Filip Wolski, Prafulla Dhariwal, Alec Radford, Oleg Klimov"
+    title:   "Proximal Policy Optimization Algorithms"
+    venue:   "Arxiv"
```

Comment:
Add the year?

Reply:
maybe add the real venue.

Reply:
I can't seem to find where it was published, everything points to Arxiv.

Reply:
ok